app/views/sessions/new.html.erb

```diff
@@ -1,13 +1,40 @@
-<%= form_for :session, url: login_path  do |f| %>
+<div class="row main signup-login-form">
+	<div class="main-login main-center">
+	<h5>Sign in ScootsMaster</h5>
+    <%= form_for :session, url: login_path  do |f|  %>
 
-  <%= f.label :username  %>
-  <%= f.text_field :username  %>
+      <div class="form-group">
+        <%= f.label :username, class: "cols-sm-2 control-label" %>
+        <div class="cols-sm-10">
+          <div class="input-group">
+            <span class="input-group-addon"><i class="fa fa-user fa" aria-hidden="true"></i></span>
+            <%= f.text_field :username, class: "form-control", placeholder: "Enter Username" %>
+          </div>
+        </div>
+      </div>
 
-  <%= f.label :password  %>
-  <%= f.password_field :password  %>
+      <div class="form-group">
+        <%= f.label :password, class: "cols-sm-2 control-label" %>
+        <div class="cols-sm-10">
+          <div class="input-group">
+            <span class="input-group-addon"><i class="fa fa-user fa" aria-hidden="true"></i></span>
+            <%= f.password_field :password, class: "form-control", placeholder: "Enter Password" %>
+          </div>
+        </div>
+      </div
 
-  <%= f.submit "Get Ur Scoot On" %>
+			<div class="form-group ">
+				<%= f.submit "Get Ur Scoot On", id: "button", class: "btn btn-success btn-lg btn-block login-button" %>
+        <%= link_to "Create Account", new_user_path, class: "btn btn-primary btn-md btn-block login-button" %>
+			</div>
 
-<% end  %>
+    <% end %>
 
-<%= link_to "Create Account", new_user_path %>
+	</div>
+</div>
+</div>
+
+<!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+<!-- Include all compiled plugins (below), or include individual files as needed -->
+<script src="js/bootstrap.min.js"></script>
```

Comment:
Same idea as with importing the bootstrap link tag, might try playing around to see if we can cut this out and still have everything work.